test/com/google/enterprise/adaptor/database/JdbcFixture.java

```diff
@@ -47,15 +53,50 @@
    * The default escape character is disabled, to match the DQL
    * behavior (and standard SQL).
    */
+  static {
+    String dbname = "h2";
+    String dbdriver = "org.h2.Driver";
+    String dburl = "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DEFAULT_ESCAPE=";
+    String dbuser = "sa";
+    String dbpassword = "";
+
+    String propertiesFile = System.getProperty("build.properties");
+    Properties properties = new Properties();
+    if (!Strings.isNullOrEmpty(propertiesFile)) {
+      try {
+        properties.load(new FileReader(propertiesFile));
```

Comment:
This uses the platform default character encoding, which is rarely desired. I would probably just use a FileInputStream, so that Properties will use Latin-1 by default, but UTF-8 is another option. Also, use try-with-resources so that the stream will be closed.